integration_tests/agent_test.go

```diff
@@ -1,4 +1,4 @@
-package agent_test
```

Comment:
Can we move it to /tests/mock-e2e/agent/  in alignment with https://github.com/flightctl/flightctl/pull/204/files if that one makes sense to you?

There, I define two make targets, but so far, we only have one, so it's good to just keep it as integration-test and run-integration test. The dirs would make it easier for me to add the non-mock target.

Reply:
Sure, Ill wait until #204 is merged